### ISSUE TITLE
Corrupt_Files (upto 2016 and in some cases upto 2015)

### ISSUE THREAD

**Issue by shashankbansal6** [TerraFusion/basicFusion#208]
The following are the Corrupt files for MODIS and ASTER. There are no corrupt files in CERES, MISR and MOPITT.

[MODIS_Corrupt_Check_MOD02HKM_2000.txt](https://github.com/TerraFusion/basicFusion/files/1136909/MODIS_Corrupt_Check_MOD02HKM_2000.txt)
[MODIS_Corrupt_Check_MOD02HKM_2001.txt](https://github.com/TerraFusion/basicFusion/files/1136911/MODIS_Corrupt_Check_MOD02HKM_2001.txt)
[MODIS_Corrupt_Check_MOD02HKM_2004.txt](https://github.com/TerraFusion/basicFusion/files/1136914/MODIS_Corrupt_Check_MOD02HKM_2004.txt)
[MODIS_Corrupt_Check_MOD02HKM_2005.txt](https://github.com/TerraFusion/basicFusion/files/1136912/MODIS_Corrupt_Check_MOD02HKM_2005.txt)
[MODIS_Corrupt_Check_MOD02HKM_2006.txt](https://github.com/TerraFusion/basicFusion/files/1136908/MODIS_Corrupt_Check_MOD02HKM_2006.txt)
[MODIS_Corrupt_Check_MOD02HKM_2007.txt](https://github.com/TerraFusion/basicFusion/files/1136910/MODIS_Corrupt_Check_MOD02HKM_2007.txt)
[MODIS_Corrupt_Check_MOD02HKM_2008.txt](https://github.com/TerraFusion/basicFusion/files/1136916/MODIS_Corrupt_Check_MOD02HKM_2008.txt)
[MODIS_Corrupt_Check_MOD02HKM_2009.txt](https://github.com/TerraFusion/basicFusion/files/1136913/MODIS_Corrupt_Check_MOD02HKM_2009.txt)
[MODIS_Corrupt_Check_MOD02HKM_2010.txt](https://github.com/TerraFusion/basicFusion/files/1136917/MODIS_Corrupt_Check_MOD02HKM_2010.txt)
[MODIS_Corrupt_Check_MOD02HKM_2011.txt](https://github.com/TerraFusion/basicFusion/files/1136918/MODIS_Corrupt_Check_MOD02HKM_2011.txt)
[MODIS_Corrupt_Check_MOD02HKM_2012.txt](https://github.com/TerraFusion/basicFusion/files/1136915/MODIS_Corrupt_Check_MOD02HKM_2012.txt)
[MODIS_Corrupt_Check_MOD02HKM_2013.txt](https://github.com/TerraFusion/basicFusion/files/1136921/MODIS_Corrupt_Check_MOD02HKM_2013.txt)
[MODIS_Corrupt_Check_MOD02HKM_2014.txt](https://github.com/TerraFusion/basicFusion/files/1136920/MODIS_Corrupt_Check_MOD02HKM_2014.txt)
[MODIS_Corrupt_Check_MOD02HKM_2015.txt](https://github.com/TerraFusion/basicFusion/files/1136919/MODIS_Corrupt_Check_MOD02HKM_2015.txt)
[MODIS_Corrupt_Check_MOD021KM_2006.txt](https://github.com/TerraFusion/basicFusion/files/1136923/MODIS_Corrupt_Check_MOD021KM_2006.txt)
[MODIS_Corrupt_Check_MOD021KM_2010.txt](https://github.com/TerraFusion/basicFusion/files/1136922/MODIS_Corrupt_Check_MOD021KM_2010.txt)
[MODIS_Corrupt_Check_MOD021KM_2012.txt](https://github.com/TerraFusion/basicFusion/files/1136924/MODIS_Corrupt_Check_MOD021KM_2012.txt)

[ASTER_Corrupt_Check_2000.txt](https://github.com/TerraFusion/basicFusion/files/1136932/ASTER_Corrupt_Check_2000.txt)
[ASTER_Corrupt_Check_2003.txt](https://github.com/TerraFusion/basicFusion/files/1136928/ASTER_Corrupt_Check_2003.txt)
[ASTER_Corrupt_Check_2004.txt](https://github.com/TerraFusion/basicFusion/files/1136930/ASTER_Corrupt_Check_2004.txt)
[ASTER_Corrupt_Check_2005.txt](https://github.com/TerraFusion/basicFusion/files/1136934/ASTER_Corrupt_Check_2005.txt)
[ASTER_Corrupt_Check_2006.txt](https://github.com/TerraFusion/basicFusion/files/1136931/ASTER_Corrupt_Check_2006.txt)
[ASTER_Corrupt_Check_2007.txt](https://github.com/TerraFusion/basicFusion/files/1136929/ASTER_Corrupt_Check_2007.txt)
[ASTER_Corrupt_Check_2008.txt](https://github.com/TerraFusion/basicFusion/files/1136933/ASTER_Corrupt_Check_2008.txt)
[ASTER_Corrupt_Check_2009.txt](https://github.com/TerraFusion/basicFusion/files/1136937/ASTER_Corrupt_Check_2009.txt)
[ASTER_Corrupt_Check_2010.txt](https://github.com/TerraFusion/basicFusion/files/1136935/ASTER_Corrupt_Check_2010.txt)
[ASTER_Corrupt_Check_2012.txt](https://github.com/TerraFusion/basicFusion/files/1136936/ASTER_Corrupt_Check_2012.txt)
[ASTER_Corrupt_Check_2013.txt](https://github.com/TerraFusion/basicFusion/files/1136938/ASTER_Corrupt_Check_2013.txt)
[ASTER_Corrupt_Check_2014.txt](https://github.com/TerraFusion/basicFusion/files/1136939/ASTER_Corrupt_Check_2014.txt)

**Comment by sandman0o0** [comment]
What is the process for your ASTER corruption check?  I think everything in those lists, has been pulled and verified aside from a few.  There were a handful of lines in those files without full filenames:

ASTER_Corrupt_Check_2006.txt:f
ASTER_Corrupt_Check_2007.txt:5_93361.hdf
ASTER_Corrupt_Check_2007.txt:520164525_47665.hdf
ASTER_Corrupt_Check_2012.txt:359_32195.hdf
ASTER_Corrupt_Check_2013.txt:479.hdf

**Comment by LandonTClipp** [comment]
We have been using the ncdump tool to determine whether the file is corrupt or not. It is a very computationally intensive process, but not a problem for computing clusters.

I will have to let @shashankbansal6 handle these bad filenames.

**Comment by sandman0o0** [comment]
since I know what year each of these is from, I can trim down the filelists and do md5sum checks.  That should be not quite as intense.  Still tedious for the number of files, but I'll start that now as well.

**Comment by sandman0o0** [comment]
I was able to get a few from 2006 that were missing.  
2007:
   AST_L1T_00303302007005923_20150518212555_93361.hdf
   AST_L1T_00307212007194247_20150520164525_47665.hdf
2012: 
   AST_L1T_00307042012080030_20150611160359_32195.hdf
The 2013 file ending in 479.hdf I have not been able to track down using my tests.  There are 212 files in 2013 that end with 479.hdf.  It also appears that most of these are no longer on the download site.  I also noticed a good chunk of 2007 is no longer available for download either.
